src/components/main/HomeScreen.js

```diff
@@ -76,10 +76,22 @@ export default class HomeScreen extends React.Component {
 
   // Check if the inputed flight number is valid
   checkNum() {
-    let chars = this.state.flight.slice(0, 2).toUpperCase();
-    console.log(chars);
-    let nums = this.state.flight.slice(2);
-    console.log(nums);
+    let charsIata = this.state.flight.slice(0, 2).toUpperCase();
```

Comment:
what happens if there's a space? Won't affect iata v acao differentiation, however could affect the API call